src/utility/RetryUtility.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export class RetryUtility {
```

Comment:
This looks quite similar to the logic in `HttpRequestHandler`. Do you have any plans to reconcile these?